### PR TITLE
Added Features to Country Map Viz Type

### PR DESCRIPTION
**Pull request by avrilpearl**
I have added more functionality to the Country Map Visualization type, specific to standard Choropleth map generation in the Geography world:

- the ability to specify the scale for the choropleth map (linear, quantize, quantile) screen shot 2017-09-01 at 2 30 51 pm
- the ability to specify the number of buckets to group your data in (3-12) screen shot 2017-09-01 at 2 34 06 pm
- the ability to display a legend screen shot 2017-09-01 at 2 35 40 pm screen shot 2017-09-01 at 2 35 49 pm
- the ability to specify the numerical format of your data for display in the hover feature and the legend 
screen shot 2017-09-01 at 2 38 08 pm screen shot 2017-09-01 at 2 37 57 pm
- a broader set of color shemes to pick from based on the mapping scales from [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3) and made sure all values have a border because now the color scheme consists of more than just 'white' for lighter values   
screen shot 2017-09-01 at 2 38 17 pm

_Note_ : As the linear color scheme also impacts the heatmap viz, the added color schemes are available in the selector for that viz type.

### CODE REVIEW

File: superset/assets/javascripts/SqlLab/components/SplitPane.jsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```
Comment (mistercrunch) [inline superset/assets/javascripts/SqlLab/components/SplitPane.jsx]: I don't think this file belongs in this PR

**Review by mistercrunch — CHANGES_REQUESTED** [review]
Remove unrelated changes

File: superset/assets/javascripts/SqlLab/components/SplitPane.jsx (hunk 1 above)
Comment (avrilpearl) [reply superset/assets/javascripts/SqlLab/components/SplitPane.jsx]: @mistercrunch i have removed this from the branch and pr

**Comment by slarrain** [comment]
Do you think this is ready to be reviewed @mistercrunch ?

**Comment by xrmx** [comment]
@slarrain quite hard to review while the PR has merge conflicts

**Comment by slarrain** [comment]
@xrmx yep, you're right Ricardo. They arose with the new version. Sorry about that.

**Comment by wildanm** [comment]
Is there any plan to merge/add this feature to the master ?

**Pull request closed without merging**